Src/BootCamp.Chapter/Lession3.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lession3
+    {
+        
+        public static void Demo()
+        {
+            bool backtotop = true;
+            while(backtotop)
+            { 
+            //User data input
+            var firstname = MessageString("What is your first name ?");
+            var secondname = MessageString("What is your second name ? ");
+            var age = MessageInt("What is your age?");
+            var weight = MessageFloat("What is your weight in kg?");
+            var height = MessageFloat("What is your height in Meters?");
+           
+            //Calc and return BMI
+            CalcBMI(weight, height);
+
+            //Display output to user
+            Console.WriteLine(firstname + " " + secondname + "is " + age + " years old, his weight is " + weight + " kg and his height is " + height + " cm ");
+            Console.WriteLine("Your BMI is " + CalcBMI(weight, height));
+
+            //Continue
+            Console.WriteLine("Would you like to do another person Y/N?");
+            var option = Console.ReadLine();
+            if ((option) == "N" || (option) == "n")
+                {
+                backtotop = false;
+                }
+
+            }
+        }
+        public static string MessageString(string message)
+        {
+            Console.WriteLine(message);
+            return Console.ReadLine();
+        }
+        public static float MessageFloat(string message)
+        {
+            return Convert.ToSingle(MessageString(message));
+        }
+        public static int MessageInt(string message)
+        {
+            return Convert.ToInt32(MessageString(message));
+        }
+
+        public static float CalcBMI(float weight, float height)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename method 'CalcBMI' to match pascal case naming rules, consider using 'CalcBmi'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4857452)